.github/workflows/common.yml

```diff
@@ -109,19 +109,22 @@ jobs:
           echo "make --version"
           make --version
           file $(which make)
+        shell: bash
 
       - name: Setup C API build
         run: |
           mkdir -p build
           cd build
           # specify C compiler as conan could not find it automatically
           # see https://github.com/conan-io/conan/issues/4322
-          cmake -DPython3_EXECUTABLE=`which python` -DCMAKE_C_COMPILER=gcc ..
+          cmake -DCMAKE_C_COMPILER=gcc ${{ env.CMAKE_GIT_EXCLUDE }} ..
```

Comment:
Lets see if this check fail after removing `Python3_EXECUTABLE` https://github.com/vsbogd/hyperon-experimental/actions/runs/13664492593

Reply:
Checked using manual CI https://github.com/vsbogd/hyperon-experimental/actions/runs/13669927540 works correctly.